wifiphisher/pywifiphisher.py

```diff
@@ -605,6 +629,12 @@ def start(self):
             time.sleep(1)
             self.stop()
 
+        if not args.no_mac_randomization:
+            print ("[{0}!{1}]" + ap_iface.get_name() + ' mac address becomes '
+                     + ap_iface.get_current_mac()).format(R, W)
+            print ("[{0}!{1}]" + mon_iface.get_name() + ' mac address becomes '
```

Comment:
I know I asked for a "[!]" but I now think that the best message to print would be:

[+] Changing the MAC address of wlan0 to 00:00:00:ac:bc:df

Don't forget the space after the "[+]".